xsd/netex_part_1/part1_ifopt/netex_ifopt_localService_support.xsd

```diff
@@ -53,7 +53,7 @@ Rail transport, Roads and Road transport
 		<xsd:documentation>LOCAL SERVICE identifier types for IFOPT Fixed Objects in Public Transport.</xsd:documentation>
 	</xsd:annotation>
 	<!-- ======================================================================= -->
-	<xsd:element name="LocalServiceRef" type="LocalServiceRefStructure" substitutionGroup="EquipmentRef">
+	<xsd:element name="LocalServiceRef" type="LocalServiceRefStructure" abstract="true" substitutionGroup="EquipmentRef">
```

Comment:
Could someone elaborate how this is related to this pull request and if it is inteded?